pxtarget.json

```
@@ -109,6 +109,8 @@
         "copyrightText": "LEGO, the LEGO logo, MINDSTORMS and the MINDSTORMS EV3 logo are trademarks and/ or copyrights of the LEGO Group. ©2018 The LEGO Group. All rights reserved.",
         "crowdinProject": "makecode",
         "selectLanguage": true,
+        "blocksCollapsing": true,
+        "highContrast": false,
```

```suggestion
        "highContrast": true,
```
Drop this one, needs to stay in; I'll try and find some time to port over a fix for whatever is causing it to do nothing, probably something simple

---

okay